utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameter.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.android.util.platformparameter
+
+/**
+ * Generic interface that is used to provide Platform Parameter Values corresponding to the
+ * [PlatformParameter] proto values. Objects that implement this interface will override the [value]
+ * property to store the actual platform parameter value.
+ * */
+interface PlatformParameter<T> {
```

Comment:
Could we rename this as PlatformParameterValue? Since it's just holding the value, and that lets us avoid naming collisions with the proto type

Reply:
Yeah that will be better. Done